test/functional/ex_cmds/trust_spec.lua

```diff
@@ -0,0 +1,67 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local eq = helpers.eq
+local clear = helpers.clear
+local command = helpers.command
+local pathsep = helpers.get_pathsep()
+local iswin = helpers.iswin()
+local curbufmeths = helpers.curbufmeths
+local exec_lua = helpers.exec_lua
+local feed_command = helpers.feed_command
+local feed = helpers.feed
+local funcs = helpers.funcs
+local pcall_err = helpers.pcall_err
+
+describe(':trust', function()
+  local xstate = 'Xstate'
+
+  setup(function()
+    helpers.mkdir_p(xstate .. pathsep .. (iswin and 'nvim-data' or 'nvim'))
+  end)
+
+  teardown(function()
+    helpers.rmdir(xstate)
+  end)
+
+  before_each(function()
+    helpers.write_file('test_file', 'test')
+  end)
+
+  after_each(function()
+    os.remove('test_file')
+  end)
+
+  it('allow then deny then forget a file', function()
+    local cwd = funcs.getcwd()
```

Comment:
It might be too late for me, because it seems like I'm missing something here, and I can't figure what is it :). This test file is very similar to `secure_spec.lua`, yet here `getcwd()` throws an error for me
```
ERROR    test/functional/ex_cmds/trust_spec.lua @ 51: :trust deny then allow then forget a file
test/functional/helpers.lua:103: attempt to index upvalue 'session' (a nil value)

stack traceback:
	test/functional/helpers.lua:103: in function 'getcwd'
	test/functional/ex_cmds/trust_spec.lua:52: in function <test/functional/ex_cmds/trust_spec.lua:51>
```

Reply:
You need a `before_each(clear)` block (see `secure_spec.lua`). The `clear` function sets up the remote Nvim process. When that process does not exist, you get the error above.

Reply:
I knew I was missing something obvious. Thank you!
Resolved in 772d0320e4065c322eafe60a955c670992f014e9